.github/workflows/publish.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
       - name: Release
         run: |
-          cargo +nightly publish -Z package-workspace --no-verify --allow-dirty --workspace \
```

Comment:
We don't need package-workspace anymore?

Reply:
```
warning: flag `-Z package-workspace` has been stabilized in the 1.89 release, and is no longer necessary
  Workspace packaging and publishing (a.k.a. `-Zpackage-workspace`) is now always enabled.
```